README.md

```diff
@@ -17,12 +17,13 @@ This means it's not possible to use anything in the [`std`](https://doc.rust-lan
 
 ## SDK version
 
-Currently supports SDK 28.2 ([flipperzero-firmware@0.84.1](https://github.com/flipperdevices/flipperzero-firmware/tree/0.84.1)).
+Currently supports SDK 34.3 ([flipperzero-firmware@0.88.0](https://github.com/flipperdevices/flipperzero-firmware/tree/0.88.0)).
 
 The crate major version number will be updated after a bump in [API version](https://github.com/flipperdevices/flipperzero-firmware/blob/release/firmware/targets/f7/api_symbols.csv) in the Flipper Zero firmware.
 
 | Crate version | API version |
-| ------------- | ----------- |
+|---------------|-------------|
+| 0.11.x        | 34.3        |
```

Comment:
Before 0.11 is released, we should not imply that a particular API version will be in that release (API 35.0 was just released in 0.89.0-rc, so it's possible we'll end up doing another bump before 0.11).
```suggestion
| Unreleased    | 34.3        |
```